### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,13 @@ To help triage or fix bugs, see the current [Ignition issues](https://github.com
 
 ## Config Validation
 
-To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases], and an online validator available [on the CoreOS website][online-validator].
+To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases]. Three is also an ignition-validate container: `quay.io/coreos/ignition-validate`. 
+
+Example:
+```
+# This example uses podman, but docker can be used too
+podman run -i quay.io/coreos/ignition-validate - < myconfig.ign
```
Comment (lucab) [inline README.md]: Minor: a `--rm` would be helpful here.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM golang:latest AS builder
```
Comment (jlebon) [inline Dockerfile]: Minor: I'd call this file e.g. `Dockerfile.validate` or something to be more clear.

File: README.md [hunk 3]
```diff
@@ -24,7 +24,13 @@ To help triage or fix bugs, see the current [Ignition issues](https://github.com
 
 ## Config Validation
 
-To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases], and an online validator available [on the CoreOS website][online-validator].
+To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases]. Three is also an ignition-validate container: `quay.io/coreos/ignition-validate`.
```
Comment (jlebon) [inline README.md]: Super minor nits: CLI, also `ignition-validate` in backticks?

File: README.md [hunk 4]
```diff
@@ -24,7 +24,13 @@ To help triage or fix bugs, see the current [Ignition issues](https://github.com
 
 ## Config Validation
 
-To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases], and an online validator available [on the CoreOS website][online-validator].
+To validate a config for Ignition there are binaries for a cli tool called `ignition-validate` available [on the releases page][releases]. Three is also an ignition-validate container: `quay.io/coreos/ignition-validate`.
```
Comment (jlebon) [inline README.md]: Sorry, totally missed this the first time: s/Three/There/  :)

Comment (ajeddeloh) [reply README.md]: Fixed

**Pull request merged**